builders/build/substrate-api/chopsticks.md

```diff
@@ -94,31 +94,25 @@ A configuration file is not necessary, however. There are additional commands an
-|         endpoint         | The endpoint of the parachain to fork.                                                              |
-|          block           | Use to specify at which block hash or number to replay the fork.                                    |
-|      wasm-override       | Path of the WASM to use as the parachain runtime, instead of an endpoint's runtime.                 |
-|            db            | Path to the name of the file that stores or will store the parachain's database.                    |
-|          config          | Path or URL of the config file.                                                                     |
-|           port           | The port to expose an endpoint on.                                                                  |
-|      build-block-mode    | How blocks should be built in the fork: batch, manual, instant.                                     |
-|      import-storage      | A pre-defined JSON/YAML storage file path to override in the parachain's storage.                   |
-| allow-unresolved-imports | Whether to allow WASM unresolved imports when using a WASM to build the parachain.                  |
-|       import-storage     | Include to generate storage diff preview between blocks.                                            |
-|    mock-signature-host   | Mock signature host so any signature starts with 0xdeadbeef and filled by 0xcd is considered valid. |
+|         endpoint         |                               The endpoint of the parachain to fork.                                |
+|          block           |                  Use to specify at which block hash or number to replay the fork.                   |
+|      wasm-override       |         Path of the WASM to use as the parachain runtime, instead of an endpoint's runtime.         |
+|            db            |          Path to the name of the file that stores or will store the parachain's database.           |
+|          config          |                                   Path or URL of the config file.                                   |
+|           port           |                                 The port to expose an endpoint on.                                  |
+|     build-block-mode     |                   How blocks should be built in the fork: batch, manual, instant.                   |
+|      import-storage      |          A pre-defined JSON/YAML storage file path to override in the parachain's storage.          |
+| allow-unresolved-imports |         Whether to allow WASM unresolved imports when using a WASM to build the parachain.          |
+|      import-storage      |                      Include to generate storage diff preview between blocks.                       |
+|   mock-signature-host    | Mock signature host so any signature starts with 0xdeadbeef and filled by 0xcd is considered valid. |
```

Comment:
Just wondering, is there a plugin to center these, or do we just do them manually?

Reply:
You could also use some VS-Code shortcuts to quickly center it. Let me do it

Reply:
Oh yeah but in this case because the table has a lot of text I think we can't do anything 😓